websocket/tests/test_app.py

```diff
@@ -191,7 +191,7 @@ def testOpcodeBinary(self):
         """ Test WebSocketApp binary opcode
         """
         # The lack of wss:// in the URL below is on purpose
```

Comment:
This comment needs changing to say the opposite, no? Including the scheme is now REQUIRED.

Reply:
good point, I'll remove the comment because it doesn't make sense. I had to comment out this test already because the URL no longer works